pyaedt/hfss3dlayout.py

```diff
@@ -707,7 +707,7 @@ def create_linear_count_sweep(
                 sweep.props["UseQ3DForDC"] = use_q3d_for_dc
                 sweep.props["MaxSolutions"] = interpolation_max_solutions
                 sweep.update()
-                self.logger.glb.add_info_message("Linear count sweep {} has been correctly created".format(sweepname))
+                self.logger.glb.info("Linear count sweep {} has been correctly created".format(sweepname))
```

Comment:
Format strings won't work on IronPython.
```suggestion
                self.logger.glb.info("Linear count sweep %s has been correctly created", sweepname)
```